integration/queries.graphql

```diff
@@ -1,5 +1,8 @@
 query LoadAuthors {
   authors {
     name
+    status {
+      name
```

Comment:
For now the "test" is just running `npm run graphql-codegen` locally and watching for the error to get reported...

Reply:
Hmm.. I was about to ask for a test regarding this. I guess this will suffice for now.